### ISSUE TITLE
Why multiplying the uncertainty by 0.5?

### ISSUE THREAD

**Issue by OmriKaduri** [oxcsaml2019/multitask-learning#2]
Hi, at your [loss function](https://github.com/oxcsaml2019/multitask-learning/blob/master/multitask-learning/losses.py#L87) you calculate:

```python
loss += torch.exp(-sem_uncertainty) * sem_loss + 0.5 * sem_uncertainty
```

Where `sem_uncertainty` should be the log variance of the semantic segmentation task, but why do you multiply it by 0.5?

As can be seen in the original paper, there is no divsion of that component:
![image](https://user-images.githubusercontent.com/7421460/98551469-03e22580-22a6-11eb-8ab0-6ca2023d96d4.png)

**Comment by 3xWangDot** [comment]
@OmriKaduri 
here is my understanding:
Assume the sigma in the original paper is x, and sigma in the code is y, then we will have log(x)=0.5*y, then x=e^(0.5*y).
After that, taking x=e^(0.5*y) into the original formulation, x^(-2) will become e^(-y), so the original formulation: x^(-2)*loss+log(x)  will become: e^(-y)*loss+0.5*y
